src/trans/internal/ftdir_mod.F90

```diff
@@ -86,7 +86,7 @@ SUBROUTINE FTDIR(PREEL,KFIELDS,KGL)
 
     IF( T%LUSEFFT992(KGL) )THEN
 
-      CALL FFT992(PREEL(1,IOFF),T%TRIGS(1,KGL),&
+      CALL FFT992(PREEL(1:,IOFF:),T%TRIGS(1,KGL),&
```

Comment:
The first argument to FFT992 is expected to be a 1-dimensional array.
Would it then not be sufficient to pass `PREEL(:,IOFF)` ?

Reply:
Yes you are correct.  I have pushed an update to the branch.